apps/server/src/controllers/ontimeController.config.ts

```diff
@@ -0,0 +1,68 @@
+export const emptyProject = {
```

Comment:
we should not introduce more variations on the source for our models
https://github.com/cpvalente/ontime/blob/master/apps/server/src/models/dataModel.ts

Reply:
oops didn't know this existed, I looked for it but apparently not enough

makes much more sense